### PR TITLE
[Serializer] Add docs for attributes context key

### PR DESCRIPTION
**Pull request by dunglas**
Documentation for https://github.com/symfony/symfony/pull/18834

### CODE REVIEW

File: components/serializer.rst [hunk 1]
```diff
@@ -333,6 +333,46 @@ You are now able to serialize only attributes in the groups you want::
...
+
+    $user = new User();
+    $user->familyName = 'Dunglas';
+    $user->givenName = 'Kévin';
+    $user->company = $company;
+
+    $serializer = new Serializer(array(new ObjectNormalizer()));
+
+    $data = $serializer->normalize($user, null, array('attributes' => array('familyName', 'company' => ['name'])));
+    // $data = array('familyName' => 'Dunglas', 'company' => ['name' => 'Les-Tilleuls.coop']);
```
Comment (javiereguiluz) [inline components/serializer.rst]: `'company' => ['name'  ...` -> `'company' => array('name'  ...`

File: components/serializer.rst [hunk 2]
```diff
@@ -333,6 +333,46 @@ You are now able to serialize only attributes in the groups you want::
...
+
+Only attributes that are not ignored (see below) are available.
+If some serialization groups are set, only attributes allowed by those groups can be used.
+
+As for groups, attributes can be applied during both the serialization and deserialization.
```
Comment (javiereguiluz) [inline components/serializer.rst]: I don't understand what this phrase means -> "As for groups, attributes can be applied during both the serialization and deserialization."

**Review by javiereguiluz — APPROVED** [review]
Perfectly explained, concise and with a good example.

Thank you!

**Comment by weaverryan** [comment]
Thanks @dunglas!

**Pull request merged**